docs/source/app-dev/app-arch.rst

```diff
@@ -148,3 +148,28 @@ To use command deduplication, you should:
 
 
 For more details on command deduplication, see the :ref:`Ledger API Services <command-submission-service-deduplication>` documentation.
+
+.. _dealing-with-time:
+
+Dealing with time
+*****************
+
+The DAML language contains a function :ref:`getTime() <daml-ref-gettime>` which returns the “current time”. The notion of time comes with a lot of problems in a distributed setting: different participants might run slightly different clocks, transactions would not be allowed to “overtake” each other during DAML interpretation, i.e., a long-running command could block all other commands, and many more.
```

Comment:
The `()` in `getTime()` is odd. Looks like it's a Java(script) function. Is that meant to be there?

Reply:
sorry, old habits die hard. Thanks.